crates/solvers/src/domain/solver/dex/mod.rs

```diff
@@ -86,42 +101,65 @@ impl Dex {
             .filter_map(future::ready)
     }
 
+    async fn try_solve(
```

Comment:
Since we don't retry until we get a non-rate-limited result we could still skip orders.
IMO this is fine but I think this is not what you wanted to avoid in the initial implementation.

Reply:
Will create a separate issue to consider using a retry mechanism to not overload this PR